tools/autograd/gen_python_functions.py

```diff
@@ -171,11 +171,15 @@
     "mul_.Scalar(Tensor(a!) self, Scalar other) -> Tensor(a!)",
     "div.Scalar(Tensor self, Scalar other) -> Tensor",
     "div_.Scalar(Tensor(a!) self, Scalar other) -> Tensor(a!)",
+    "_fused_moving_avg_obs_fq_helper.functional(Tensor self, Tensor observer_on, Tensor fake_quant_on, Tensor running_min, Tensor running_max, Tensor scale, Tensor zero_point, float averaging_const, int quant_min, int quant_max, int ch_axis, bool per_row_fake_quant=False, bool symmetric_quant=False) -> (Tensor output, Tensor mask, Tensor running_min, Tensor running_max, Tensor scale, Tensor zero_point)",  # only used by the functionalization pass # noqa: B950
```

Comment:
Why isn't this one already caught by the tag check below?

Reply:
It is and I should delete this, thanks. Relic of the original version of the PR (where I manually implemented this operator)